### ISSUE TITLE
🎉 Outreachy Winter 2022: Applicants tasks/contributions: Localisation

### ISSUE THREAD

**Issue by BatoolMM** [Open-Science-Community-Saudi-Arabia/localisation#13]
## 🎊 Welcome Outreachy 2022 Contributors! We're excited that you're here 🎉

>This ticket will serve as the central point tracking the contributions of perspective Outreachy (spring 2022 cohort) applicants.

Outreachy provides paid remote, three-month internships to support diversity in Free and Open Source Software (FOSS). For more information about what Outreachy is, follow [this link](https://www.outreachy.org/).

**Outreachy internships are:**

💵 Paid - $7,000 USD total internship stipend

🌍 Remote - both interns and mentors work remotely

🕰 3 months - internships run May to August.

For those interested, this process is described [here](https://www.outreachy.org/apply/make-contributions/).

> We ask all outreachy candidates to work **on a minimum of 3 projects** simultaneously so that they are not completely relying on their selection in OSCSA if we end up not selecting them!

## Summary

Open Science Community Saudi Arabia (OSCSA) aims to introduce open educational resources (OERs) for Open Science in Arabic to facilitate learning of Open science practices to novice learners. Therefore, we work with other Open Science communities such as the Turing Way and Open Life Science to localise their open educational resources (OER) using [Crowdin](https://crowdin.com/), GitHub and Zenodo.

We use GitHub as a version control system for all the translations and to acknowledge all contributors. Crowdin is used as a translation management system (or TMS) to make contributions smooth. Any changes made to Crowdin will be reflected on GitHub using automation and continuous integration (CI). Crowdin provides Translation Memory (TM), Glossary, Machine Translation (MT) and QA checks.

Open Science Community Saudi Arabia (OSCSA) works with other international and local communities to reuse and curate their materials. We have so far worked with:

- Open Life Science (OLS) for Translation for Open Life Science Talks and Transcripts.
> The open Life Science (OLS) mentoring and training program has been running since January 2020 and offers a 16-week project-based opportunity to learn open science practices. To make some of its resources more accessible, [Talarify](https://twitter.com/talarify?lang=en) and [Open Science Community in Saudi Arabia](https://twitter.com/OpenSciSaudi) translated part of OLS3 cohort resources. More details can be found in [this respository](https://github.com/Open-Science-Community-Saudi-Arabia/ols3-cohort-talks-and-transcripts).

-  Translation of The Turing Way
> The Turing Way is a handbook to reproducible, ethical and collaborative data science. It supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone. Part of the topics in _the Turing Way_ was already translated to Spanish, OSCSA has introduced the same workflow to translate the Turing Way to Arabic. 

In Outreachy Winter 2022, we are accepting applicants to localise specific chapters of  [The Turing Way](https://the-turing-way.netlify.app/welcome.html), audit the translation of specific videos of [The open Life Science (OLS)](https://openlifesci.org/) and curate it with other materials from [the Open Science Community Saudi Arabia](https://zenodo.org/communities/1231231664/?page=1&size=20) to produce a short MOOCs that introduce Open Science for the novice learners.

Successful candidates will improve the streamlining of the localisation process by:

- Complete the localisation of specific part of the Turing Way
- Improve the localisation of the images/illustrations 
- Audit the Translation of the Open Life Sciences Videos
- Localise specific Talks and materials from Open Science Community Saudi Arabia and curate the materials in MOOCs for Open Science movie learner
- Add milestones (e.g. check boxes + progress) to track the localisation across 5 different languages 
- Improve the on-boarding and off-boarding of the localisation

### How to start?

- Join OSCSA community [here](https://form.jotform.com/222925373492360)
- Join [Crowdin](https://crowdin.com/project/ols3)

> You might need to create an account in Crowdin and join [our project here](https://turingway.crowdin.com/u/signup?hash=caa0e3045741565a01837bdb3245c03294). Once you join, you have to comment on this issue, so we follow your work in Crowdin.

- Follow the [contribution guidelines](https://github.com/open-life-science/ols3-cohort-talks-and-transcripts/blob/main/CONTRIBUTING%E2%80%8B.md).

- This way, it will open the crowdin editor where you will so all the translations! All the translation was done using crowdin machine translation (similar to google translation). None of it was done manually, so all the translation needs to be re-translated and then proofread (Approved). Chose only one single file and read it from [the TW book](https://the-turing-way.netlify.app/) before starting the translation - and start from the top of the file (title).
<img width="1433" alt="Screenshot 2022-03-28 at 19 57 07" src="https://user-images.githubusercontent.com/53487593/160449211-678c3a71-b184-4d65-930e-b3ef1db4b02b.png">

- Follow up the steps of how to use Crowdin Editor [here](https://github.com/Open-Science-Community-Saudi-Arabia/ols3-cohort-talks-and-transcripts/blob/main/CONTRIBUTING%E2%80%8B.md).

Using the Main menu in the top left corner, you can switch between files for translations, change translation language, contact manager, switch the view, get help or quit the editor. For more information, follow [the docs here](https://support.crowdin.com/online-editor/). If you have questions, please add them to this issue!

> You need only to translate one small file. You don't only translate strings, but you also have to add terms to [the glossary](https://support.crowdin.com/glossary/). 

### Glossary

The **Glossary** is supposed to contain **terminology that has a purpose of explaining some specific terms** or the ones often used in the project to be translated properly and consistently such as (e.g. open science, open access, open hardware, ...etc). It's not supposed to include names of people, books, ...etc. 

**You should add a specific term add to the glossary if:**
1- It's commonly used in the project, but translators can translate it differently. For example, open access can be translated الوصول الحر أو الوصول المفتوح أو النفاذ الحر  but we want to keep consistency throughout the project when it's translated by many contributors - so they can use the same translation, not multiple ones.
2- Some English terminology doesn't exist in Arabic. Once you translate them, you include them in Glossary (e.g. Git, GitHub, repository,..etc).

The Glossary should be consistent in style. The Arabic translation should contain the Arabic translation followed on the left the English terms between brackets with spaces between brackets. For example, the Arabic for Open access should be (Open Access) الوصول الحر with space between brackets and the Arabic word.

**If you have questions about the glossary،, ask under this issue.**

### Submission

Once you finish a very small part of the translation, submit a PR for a markdown file called `NAME_docs.md` in `/docs` in this repo to include brief documentation in **Arabic** ( around 500 words or more using Markdown) about the translation process of the project in Crowdin. This should include (Getting started with Crowdin, tips, ..etc) using images and GIFs. 

**If you have any questions throughout the process, add your question to this issue!**

- [Record in-progress contributions on the Outreachy website](https://www.outreachy.org/apply/project-selection/). You'll need to [go to the community landing page linked from the projects listing page](https://www.outreachy.org/apply/project-selection/). Then click the 'Record Contribution' button. Add links and descriptions to your contributions and the project timeline before the application deadline. You can always go back and edit your recorded contributions. A contribution doesn't have to be merged or accepted to be recorded.

### Tasks for Intern Selection:
- Complete the translation of one of Video in OLS3.
- Open PR in this repository to include brief documentation in **Arabic** (400 words or more) about the translation process of the project in Crowdin.

### Meetings

We will lead an online video call (on Zoom) to answer any questions you have. This will be on **Tuesday, the 18th of October, at 12:10 pm UTC**!

> The link for Zoom Meeting
> https://liverpool-ac-uk.zoom.us/j/93627968243?pwd=OVh5d1IreFRnOVRpNDVXMWRwMmlmZz09

### Assessing Candidates

- Candidate should be comfortable collaborating through GitHub
- Candidates should complete all tasks listed for intern selection.

### Resources
- Some documentation about the localisation of _The Turing Way_ can be found [here](https://deploy-preview-2202--the-turing-way.netlify.app/community-handbook/translation.html).
- More information about the workflow is presented [in this talk in Creative Commons Summit 2021](https://zenodo.org/record/5521150#.YjjYpZpBy3I).

**Comment by Khadeeejah** [comment]
hi @BatoolMM   i am currently translating  the documents under Reproducible Research

**Comment by BatoolMM** [comment]
@Khadeeejah reproducible research is almost completed audited. Can you try audit the translation for the videos (e.g. Open Hardware) from here: https://crowdin.com/project/ols3

**Comment by Khadeeejah** [comment]
Okay I'll do that @BatoolMM  thank you
i cant translate unless i join the team, i just sent a request to do so 
kindly accept

**Comment by Khadeeejah** [comment]
Good morning @BatoolMM the files in the open hardware folder are showing green checks and shows translated by you a year ago, but i am unable to see the remaining files that have not been transcribed, is it possible for us to jump on a short meeting at a convinient time for you for me to have a proper sync, as i have read through the docs and need some clarity, Thank you for your time.

**Comment by rnabage** [comment]
Hello @BatoolMM Thank you for accepting my request to join the Crowdin team. It seems all the files have been 100% translated, to be clear is it approving/auditing the translations you want us to work on?

**Comment by BatoolMM** [comment]
Welcome @rnabage, yes, only approving/auditing the translations is what we are looking for now! Indicate in the issue which file are you working on!

Looking forward to see the auditing!

**Comment by rnabage** [comment]
Thank you @BatoolMM . I'm be working on the 'REDME-for-Open-Projects' file. I think there has been a typo in the title of the file which should be 'README-for-Open-Projects' but I don't know how to edit a file name on Crowdin.

**Comment by BatoolMM** [comment]
Hi @rnabage, don't worry about the file name. Looking forward to see your work!

**Comment by rnabage** [comment]
Thank you @BatoolMM . Thank you for being so helpful I'm loving the work I'm doing on the project.

**Comment by Khadeeejah** [comment]
Hello @BatoolMM  are there any other task to work on under this project, I'm happy to work on it

**Comment by xiaodaiweiwei** [comment]
Hello @BatoolMM I'm an Outreachy applicant. I am looking forward to making contributions to issues.  Could I get some tasks to work on? Thank you!

**Comment by Khadeeejah** [comment]
> Hello @BatoolMM I'm an Outreachy applicant. I am looking forward to making contributions to issues.  Could I get some tasks to work on? Thank you!


Hi , read through the readme and contribution.md file for instructions on how to get started and setup your crowdin account.

**Comment by Khadeeejah** [comment]
Good day ma @BatoolMM  where do we get the link to the zoom meeting happening today

**Comment by BatoolMM** [comment]
Welcome @xiaodaiweiwei, this task is for individuals who are fluent in Arabic but you are welcome to work on other tasks in https://github.com/Open-Science-Community-Saudi-Arabia#12 but there is an open issue [here](https://github.com/open-life-science/ols3-cohort-talks-and-transcripts/issues), which you are welcome to work on it with @Khadeeejah .

@Khadeeejah You are welcome to work on other videos or the issue mentioned. We were supposed to have a Zoom session for Q/As today at 12:10pm UTC, but we had to postpone it till Tuesday at the same time. This was outside our control, but we are happy to answer questions regarding anything. We are also reviewing all your wonderful PRs. All of your contributions will be recorded and acknowledged in our README file. Thank you again for you all your amazing work 🌷

**Comment by rnabage** [comment]
Hello @BatoolMM and @Khadeeejah , this is my first time contributing to open source so I'm a bit stuck on how to create the pull request with the documentation on how to use Crowdin and I want to be sure I'm doing it right. I created a fork of this repository and created the ruqayyah_docs.md file and added just the title. I tried clicking create pull request after this to test out and try to see if it'll appear under the pull requests of this repository but it didn't appear. When I'm done adding the documentation please how do I make it appear in the pull requests under this issue? Thank you.

**Comment by Khadeeejah** [comment]
@rnabage  I can put you through on how to go about it please send me a mail , so that I can setup a Google meet with you (wuraola81@gmail.com)

**Comment by rnabage** [comment]
> @rnabage I can put you through on how to go about it please send me a mail , so that I can setup a Google meet with you ([wuraola81@gmail.com](mailto:wuraola81@gmail.com))

Okay, thank you

**Comment by BatoolMM** [comment]
@rnabage Thank you for your work, no worries, a few people also struggles with making a PR. [This video go through the steps](https://www.youtube.com/watch?v=IqJWDTZdQG4), let me know if you are still struggling

**Comment by rnabage** [comment]
Thank you @BatoolMM ! I've managed to create the pull request.

**Comment by rnabage** [comment]
Hello @BatoolMM , Good evening. I've finished translating the file I've been working on, if you could please review and revert? Also, please how do I record my work on Crowdin as a contribution on the Outreachy page?

**Comment by BatoolMM** [comment]
Thank you @rnabage, we will review your PR. You should log in to Outreachy and fill in the questions about your contribution, then your name will appear to us! Let me know if you still struggling!

**Comment by Rubadel** [comment]
@BatoolMM Hi dear, I want to contribute with this project of this Task, I sent a Join request on Crowdin could you accept me. I am Ruba Adel there.

**Comment by Khadeeejah** [comment]
Hi @BatoolMM what time is the meeting today and can we get the link to the meeting

**Comment by rnabage** [comment]
> Welcome @xiaodaiweiwei, this task is for individuals who are fluent in Arabic but you are welcome to work on other tasks in https://github.com/Open-Science-Community-Saudi-Arabia#12 but there is an open issue [here](https://github.com/open-life-science/ols3-cohort-talks-and-transcripts/issues), which you are welcome to work on it with @Khadeeejah .
> 
> @Khadeeejah You are welcome to work on other videos or the issue mentioned. We were supposed to have a Zoom session for Q/As today at 12:10pm UTC, but we had to postpone it till Tuesday at the same time. This was outside our control, but we are happy to answer questions regarding anything. We are also reviewing all your wonderful PRs. All of your contributions will be recorded and acknowledged in our README file. Thank you again for you all your amazing work 🌷

Hey @BatoolMM in this comment you mentioned that the meeting is today by 12:10pm UTC, could we have the link to the meeting please? Or has there been a new development? Thank you.

**Comment by BatoolMM** [comment]
Welcome @Rubadel to OSCSA, you were added to Crowdin. Le me know if you have any questions!

@Khadeeejah @rnabage I am sorry you couldn’t join the meeting today, just saw your messages. This is the [Slides](https://docs.google.com/presentation/d/1FsIJQBNnmAa5afUMGXoQ0xvTaSF7aSSCmHz_W8jeTn4/edit#slide=id.g162b4a8e627_0_2) we used during the meeting.

Let me know if you have any questions!

**Comment by Khadeeejah** [comment]
Hi @BatoolMM  the link is pointing to the zoom meeting and not slides, kindly check

**Comment by BatoolMM** [comment]
Thank you for catching it up, it's fixed now!

**Comment by Khadeeejah** [comment]
> @BatoolMM Hi dear, I want to contribute with this project of this Task, I sent a Join request on Crowdin could you accept me. I am Ruba Adel there.

hi @Rubadel  if you need any assistance on how to use crowdin and make a pr i am happy to assist you.

**Comment by ThanaMeri** [comment]
Hi @BatoolMM. I'm an Outreachy applicant. will start working on the task.

**Comment by BatoolMM** [comment]
Welcome @ThanaMeri 🌷

**Comment by ThanaMeri** [comment]
I will start working on Open-Data file. @BatoolMM

**Comment by Rubadel** [comment]
Hello @BatoolMM Could I ask about what I am doing, because I need to be sure if it is right..

I have join in the project in the crowding, and select the language I want then in this page

[<img width="912" alt="Screenshot 1444-04-04 at 10 12 35 PM" src="https://user-images.githubusercontent.com/66481905/198849001-b3f29465-4cab-4b4a-810c-24139ddeab36.png">](https://crowdin.com/project/ols3/ar)

I select a file and go through vote for the best suggestion of the translation or add new one is better than of the existing translation for each sentence of the video.. Is it right?

**Comment by Khadeeejah** [comment]
> Hello @BatoolMM Could I ask about what I am doing, because I need to be sure if it is right..
> 
> 
> 
> I have join in the project in the crowding, and select the language I want then in this page
> 
> 
> 
> [<img width="912" alt="Screenshot 1444-04-04 at 10 12 35 PM" src="https://user-images.githubusercontent.com/66481905/198849001-b3f29465-4cab-4b4a-810c-24139ddeab36.png">](https://crowdin.com/project/ols3/ar)
> 
> 
> 
> I select a file and go through vote for the best suggestion of the translation or add new one is better than of the existing translation for each sentence of the video.. Is it right? 

Hi, yes you are on the right path, you just have to audit/ approve the translation, see if there are error too.

**Comment by Rubadel** [comment]
well, Thank you Khadeeejah 
So now how can I make a PR for my activity

**Comment by Khadeeejah** [comment]
> well, Thank you Khadeeejah 
> 
> So now how can I make a PR for my activity 

Fork and clone the repo, create a file called (yourname.docs) inside the docs folder, you will see how the rest of us did it. Then document your contribution process in Arabic.

You can refer to the instructions in the repo to understand better. 
When you are done push to your forked repo and make a pull request 

I'm using my phone to respond to you, that's why it's hard to reference the link in this comment.

I hope I was able to help☺️✨

**Comment by Rubadel** [comment]
Thank you very much @Khadeeejah,
You are actually🌸 .. I am grateful for your helping

**Comment by BatoolMM** [comment]
Thank you @Khadeeejah for helping out and guiding the interns. 

Also, thank you all for your contributions so far. Don't worry if your PR isn't merged yet, still don't forget to add your contributions to Outreachy. In terms of the timeline, feel free to submit your your own timeline, this will not be taken into account during the selection process in OSCSA.

Outside Outreachy, we are happy for your continued involvement in the Community and you are very welcome to continue to work on any of these issues after the contribution phase. All contributors will be acknowledged and added to the GitHub README and any talk presented by community members. OSCSA is alos happy to :

- Add a recommendation on the applicant's LinkedIn profile if they provided continued involvement with the Community
- Endorse you for skills on LinkedIn
- Coaching you on how to highlight their contributions on your resume
- We also happy to provide a reference or recommendation letter
- Thank you again for all your wonderful contribution so far 🌷!

**Comment by BatoolMM** [comment]
@all-contributors please add @Khadeeejah for Mentoring

**Comment by Khadeeejah** [comment]
Hello ma @BatoolMM how are you doing, I didn't make it to the last stage of the internship and I'll appreciate it if you can let me  keep contributing on this project as it doesn't have any outreachy interns selected for it.

Thank you

**Comment by ThanaMeri** [comment]
I wonder why they didn't choose any of us for this part of the project.

**Comment by Khadeeejah** [comment]
> I wonder why they didn't choose any of us for this part of the project.

https://www.outreachy.org/docs/applicant/#estimated-number-of-interns


I guess they didn't have enough budget for both projects

But we can still keep contributing and join the community because I feel we can achieve a lot on this project

**Comment by BatoolMM** [comment]
Hi @ThanaMeri, @Khadeeejah, @rnabage, @Ruba 🌷  

Our Community can't thank you enough for your wonderful contributions, unfortunately, we couldn’t choose an intern for both projects due to a limited budget - this was a tough decision that came after a long discussion with all co-mentors. 

However, all our projects are open source and have active contributions from outside Outreachy on a voluntary basis, and you are very welcome to join these meetings and develop your skills further, if this looks interesting to you. 

@Khadeeejah thank you for showing enthusiasm, I will follow up with an email and add you to the translation channel in Slack.

**Comment by Khadeeejah** [comment]
> Hi @ThanaMeri, @Khadeeejah, @rnabage, @Ruba 🌷  
> 
> 
> 
> Our Community can't thank you enough for your wonderful contributions, unfortunately, we couldn’t choose an intern for both projects due to a limited budget - this was a tough decision that came after a long discussion with all co-mentors. 
> 
> 
> 
> However, all our projects are open source and have active contributions from outside Outreachy on a voluntary basis, and you are very welcome to join these meetings and develop your skills further, if this looks interesting to you. 
> 
> 
> 
> @Khadeeejah thank you for showing enthusiasm, I will follow up with an email and add you to the translation channel in Slack.

Thank you ma❤️

I'll be expecting
